source/agora/network/Client.d

```diff
@@ -560,8 +565,8 @@ public class NetworkClient
 
                         try
                         {
-                            this.log.format(log_level, "Request '{}' to {} failed: {}",
-                                name, conn.address, ex.message);
+                            this.log.format(log_level, "Client.attemptRequest '{}' to {}: {}/{} FAILED",
+                                name, idx +1, this.max_retries, conn.address, ex.message);
```

Comment:
Please check the order of the parameters.

Reply:
Also updated